src/renderRepoCard.js

```diff
@@ -101,6 +111,9 @@ const renderRepoCard = (repo, options = {}) => {
       ${archiveBadge}
 
       <text x="50" y="38" class="header">${header}</text>
+
+      ${isTemplate ? template: ""}
```

Comment:
You can just directly fallback to empty string on the templateBadge variable directly.

Reply:
done. But not sure the `archiveBadge` will be proper aligned. I haven't investigated that